how-to-guides/present-cross-validation-results/scripts/neptune_cross_validation.py

```diff
@@ -0,0 +1,110 @@
+# How to present CV with Neptune
+import neptune.new as neptune
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.utils.data import SubsetRandomSampler, DataLoader
+from sklearn.model_selection import KFold
+from statistics import mean
+
+# Step 1: Create a Neptune Run
+run = neptune.init(
+    project="common/showroom", tags="cross-validation", api_token="ANONYMOUS"
+)
+
+# Step 2: Log config and hyperparameters
+parameters = {
+    "epochs": 1,
+    "lr": 1e-2,
+    "bs": 10,
+    "input_sz": 32 * 32 * 3,
+    "n_classes": 10,
+    "k_folds": 2,
+    "model_name": "checkpoint.pth",
+    "device": torch.device("cuda:0" if torch.cuda.is_available() else "cpu"),
```

Comment:
pls not mix defining parameter with computing it. device methods should be outside dict.

Reply:
I think of the device as a parameter to be logged so you easily re-instantiate your model into the right device in case you want to reproduce an experiment as you would do with the seed value. 

But this is only an example so it's fine to me to leave it out 👍